examples/hand_pose_estimation/hand_keypoints_loader.py

```diff
@@ -0,0 +1,58 @@
+import glob
+import pickle
+
+from paz.abstract import Loader
+from paz.backend.image.opencv_image import load_image, resize_image
+
+
+class HandDataset(Loader):
+    def __init__(self, path, split='training', image_size=(256, 256, 3)):
+        super().__init__(path, split, None, 'HandSegmentation')
+        self.path = path
+        self.split = split
+        self.image_size = image_size
+
+    def _load_images(self, image_path):
+        image = load_image(image_path)
+        hand = resize_image(image, (self.image_size[0], self.image_size[1]))
+        return hand
+
+    def _to_list_of_dictionaries(self, hands, seg_labels=None,
+                                 annotations=None):
```

Comment:
What is the difference between "seg_labels" and "annotations"? A more common word is "labels".